Merkll/Quick-Credit
# #165971366 Improve on the User Interface Design

#### What does this PR do?
Improve UI design
#### Description of Task to be completed?
* Change UI color scheme
* Create an Admin dashboard page

#### How should this be manually tested?
* Clone this repo.
* Ensure you are on this branch
* Browse to the UI
#### Any background context you want to provide?
none

#### Screenshots (if appropriate)
![Screenshot from 2019-05-16 13-12-39](https://user-images.githubusercontent.com/34121552/57852941-67d47c80-77dc-11e9-820d-5822df48a2f9.png)
![Screenshot from 2019-05-16 13-12-16](https://user-images.githubusercontent.com/34121552/57852942-67d47c80-77dc-11e9-9f8f-e33fe6558ca5.png)
![Screenshot from 2019-05-16 13-11-52](https://user-images.githubusercontent.com/34121552/57852943-67d47c80-77dc-11e9-9069-a82bdd475512.png)
#### Questions:
